internal/pkg/cli/init.go

```diff
@@ -274,6 +283,7 @@ func BuildInitCmd() *cobra.Command {
 			return nil
 		}),
 	}
+	cmd.Flags().StringVar(&vars.profile, profileFlag, "", profileFlagDescription)
```

Comment:
Would it be nice to give default value for the flag as "default" so that we can remove the `initEnv.Ask()` and simplify the workflow?

Reply:
That's a good idea, we can set that default in `init` but leave it empty in `env init`

Reply:
But if we set the default value for the flag to be "default" then in the `initEnv.Ask()` it won't ask user to select since `o.EnvProfile` will not be an empty string.

Reply:
I can make this change, but I'd like to make sure we document it somewhere. Is there a good spot that we can stick a line in the env init calls that will remind the user "hey, this will deploy to your default profile; if that's not cool, choose no and then run env deploy later.

Reply:
I could simply modify the initShouldDeployPrompt to the following: 
`"Would you like to deploy a test environment? (This will deploy to your default profile if not otherwise specified with the --profile flag)"` but that's not dynamic, and it seems like we could do better by only printing off a warning line when the user hasn't specified a profile.